core/router.py

```
@@ -1,9 +1,17 @@
 """Routes messages between clients and Pipecat."""
 
+from collections import deque
+from typing import Deque, Dict
```

Ruff (0.16.3)

[warning] `typing.Deque` is deprecated, use `collections.deque` instead

(UP035)